rpc/gnmi/mixed-schema.md

```diff
@@ -99,6 +96,35 @@ constrained to the specified origin. Particularly:
   the specified `origin`. To delete contents from multiple origins, a client
   MUST specify multiple paths within the `delete` of the `SetRequest`.
 
+### Special considerations for the use of Origin CLI
```

Comment:
Since `cli` origin is quite special, should we provide guidance on how it is meant to be used outside of SetRequest which is described below?

What happens when `cli` origin is used in a get request? I'd assume that for NOSes that maintain different datastores a GetRequest should only contain CLI commands that can be executed from the  equivalent of a `state` datastore.
But what if someone uses a GetRequest with configuration CLI commands inside of it?

Any guidance on what should be returned back if CLI origin is used in a Subscribe request?

Reply:
Also I see the following sentence

```
Where the origin field is specified as cli, the path should be ignored and the value specified within the Update considered as CLI input.
```

which implies `cli` origin can only be used in a Set Requests. If that is true, then maybe we should put an accent on that.

Reply:
I am open to feedback on the topic of getRequest for origin CLI.  My first thought is a getRequest for origin CLI should return the complete config in the vendor's chosen 'CLI' format.

Reply:
Arista supports getRequest with cli origin as a way to retrieve state. Operators can use this a way to inspect cli configuration, or any other cli state.

For example:
```
gnmi  get origin=cli "show version"
/show version:
{
  "version": "1.2345"
}
```

Reply:
> I am open to feedback on the topic of getRequest for origin CLI. My first thought is a getRequest for origin CLI should return the complete config in the vendor's chosen 'CLI' format.

oh, I like that. e.g. get with origin `cli` and empty path returns a complete config in the CLI format, and if a path is specified, if the NOS supports it, it returns the CLI config of a path subtree

Reply:
+1 to @hellt 

The topic of this document however doesn't quite match with this information.  If @sebsapa interpretation is not intuitive and/or we feel the need to make it explicit, perhaps we should start a separate issue/PR.